PZ/WebClient/Connector.py

```diff
@@ -0,0 +1,102 @@
+import requests
+import json
+import urllib.parse
+
+from .models import *
+
+class Connector(object):
+    def __init__(self, url_adr):
+        self.url_adr = url_adr
```

Comment:
add underscore `_` before all members to make them private
`self._url_address = url_address`